docs/whats-new.md

```diff
@@ -1,36 +1,62 @@
 # What's New
 
-## [v0.40.1](https://github.com/project-stacker/stacker/releases/tag/v0.40.1)
+## [v1.0.0](https://github.com/project-stacker/stacker/releases/tag/v1.0.0-rc9)
+
+### Convert a Dockerfile for stacker
+
+- A new [`stacker convert`](reference/stacker_cli.md#stacker-convert) command performs a conversion of a Dockerfile into a stacker.yaml file. During the conversion, some variables from the Dockerfile may be exported to a substitution file.  
+
+    :pencil2: The conversion is a best-effort process and may not be successful in all cases.
+
+### Publish specific layers
+
+- By default, the [`stacker publish`](reference/stacker_cli.md#stacker-publish) command pushes all layers in a stacker.yaml file instead of only the required layers. Using a new command option, `layer <value>`, you can explicitly specify which layers are to be published.  This command option can be specified multiple times, selecting each layer to be included.
+
+### Specify a single working directory
+
+- A new [`stacker`](reference/stacker_cli.md#stacker) command option, `work-dir`, sets the working directory for stacker's cache, OCI output, and rootfs output. The existing command options `stacker-dir`, `oci-dir`, and `roots-dir` can then be omitted or used to override the `work-dir` setting.
+
+### Import contents when no shell exists
 
-* Support for `scratch`
+- Import directives can include destination paths. This feature is useful when images are built from a scratch layer before any shell has been invoked. Prior to this, a `run:` section was required to invoke a shell and to explicitly copy files to be imported into the image. For example, you can now write a directive such as the following, with no `run:` section:
```

Comment:
```suggestion
- Import directives can include destination paths. This feature is useful to simplify `run` section scripts, and for when images are built from a scratch base image, which does not contain a shell to run the script in a `run` section. Prior to this, a `run:` section was required to invoke a shell and to explicitly copy files to be imported into the image. For example, you can now write a directive such as the following, with no `run:` section:
```

Reply:
On a second reading, maybe the best way to explain this is "when images are built without a base image" instead of "a scratch base image"

@rchincha / @hallyn what do you guys think?

Reply:
Does this work?  "This feature is useful to simplify `run` section scripts, and for when images are built without a base image. With no base image, there is no shell to run the script in a `run` section."

Reply:
sounds good to me, thanks